q2_fmt/plugin_setup.py

```diff
@@ -299,5 +300,48 @@
     }
 )
 
+plugin.methods.register_function(
+    function=q2_fmt.peds_bootstrap,
```

Comment:
Did we talk about whether this is technically a bootstrap test? I think technically it's a Monte Carlo simulation. If you haven't already, can you do some reading on this to make sure we're referring to this by the correct name (and adjust the name if we're not).